### PR TITLE
Update master (v202104.1.1)

### PR DESCRIPTION
**Pull request by MakisH**
This releases the following important fixes:

- https://github.com/precice/tutorials/issues/211: openfoam_remove_empty_dirs for compressed results
- https://github.com/precice/tutorials/pull/209: small elastic-tube-1d documentation and code fixes

@uekerman or @IshaanDesai  please approve if you are fine releasing these changes as a bugfix release now.

### CODE REVIEW

File: elastic-tube-1d/README.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ The following parameters have been chosen:
 - Fluid density: $$ \rho = 1 $$
 - Young modulus: E = 10000
 
+Additionally the solvers use the parameters `N = 100`, `tau = 0.01`, `kappa = 100` by default. These values can be modified directly in each solver.
```
Comment (uekerman) [inline elastic-tube-1d/README.md:28]:
```suggestion
Additionally the solvers use the parameters `N = 100` (number of cells), `tau = 0.01` (dimensionless timestep size), `kappa = 100` (dimensionless structural stiffness) by default. These values can be modified directly in each solver.
```

**Review by IshaanDesai — APPROVED** [review]
The changes to `elastic-tube-1d` are especially relevant and hence a bug-fix release would be good.

File: multiple-perpendicular-flaps/README.md [hunk 2]
```diff
@@ -112,3 +112,5 @@ After the simulation has finished, you can visualize your results using e.g. Par
 ## References
 
 [1] H. Bungartz, F. Linder, M. Mehl, B. Uekerman. A plug-and-play coupling approach for parallel multi-field simulations. 2014.
```
Comment (uekerman) [inline multiple-perpendicular-flaps/README.md:114]:
```suggestion
[1] H. Bungartz, F. Linder, M. Mehl, B. Uekermann. A plug-and-play coupling approach for parallel multi-field simulations. _Comput Mech_ **55**, 1119-1129 (2015). https://doi.org/10.1007/s00466-014-1113-2
```

**Review by uekerman — APPROVED** [review]
I did not test the changes, but read over them. Minor tweaks below.

**Pull request merged**